examples/reagent/src/cljs/todomvc/actions.cljs

```diff
@@ -0,0 +1,39 @@
+(ns todomvc.actions
+  (:require [todomvc.session :as session]
```

Comment:
Yeah. That's why I asked it. Though we normally prefer tabs, it might be a community best-practice, then I guess we're fine with it.